### PR TITLE
fix(overview): update quick start links and intro

### PR DESCRIPTION
**Pull request by kmova**
The OpenEBS install is under-going refactoring to make
each engine installed separately. Updated the quick
start links for different engines.

Also removed Mayastor from alpha stage.

Signed-off-by: kmova <kiran.mova@mayadata.io>

### CODE REVIEW

File: docs/overview.md [hunk 1]
```diff
@@ -5,90 +5,77 @@ sidebar_label: Overview
...
+OpenEBS is a collection Storage Engines, allowing you to pick the right storage solution for your Stateful workloads and the type of Kubernetes platform.  At a high-level, OpenEBS supports two broad categories of volumes - Local and Replicated. 
+
+### Local Volumes 
 
-- When using synchronous replication, iSCSI is used to attach storage from OpenEBS to 
-  application pods. Hence OpenEBS requires iSCSI client to be configured and `iscsid` service
-  running on the worker nodes.
-  Verify if [iSCSI service is up](/docs/next/prerequisites.html) and
-  running before starting the installation.
+Local Volumes are accessible only from a single node in the cluster. Pods using Local Volume have to be scheduled on the node where volume is provisioned. Local Volumes are typically preferred for distributed workloads like Cassandra, MongoDB, Elastic, etc that are distributed in nature and have high availiability built into them.
```
Comment (ranjithwingrider) [inline docs/overview.md:30]: availiability  -> availability

File: docs/overview.md [hunk 2]
```diff
@@ -5,90 +5,77 @@ sidebar_label: Overview
...
 
-- When using synchronous replication, iSCSI is used to attach storage from OpenEBS to 
-  application pods. Hence OpenEBS requires iSCSI client to be configured and `iscsid` service
-  running on the worker nodes.
-  Verify if [iSCSI service is up](/docs/next/prerequisites.html) and
-  running before starting the installation.
+Local Volumes are accessible only from a single node in the cluster. Pods using Local Volume have to be scheduled on the node where volume is provisioned. Local Volumes are typically preferred for distributed workloads like Cassandra, MongoDB, Elastic, etc that are distributed in nature and have high availiability built into them. 
 
-- Default installation works in most of the cases. As a *Kubernetes cluster-admin*, start the default installation using either
+Depending on the type of storage attached to your Kubernetes worker nodes, you can select from Dynamic Local PV - Hostpath, Device, ZFS or Rawfile.
```
Comment (ranjithwingrider) [inline docs/overview.md:32]: you can select from Dynamic Local PV - > you can select different flavors of Dynamic Local PV such as 

It's just a suggestion.

File: docs/overview.md [hunk 3]
```diff
@@ -5,90 +5,77 @@ sidebar_label: Overview
...
 <br>
 
-## OpenEBS Storage Engines
+## Quickstart
 
-OpenEBS is a Kubernetes native hyperconverged storage solution. 
-OpenEBS consumes the storage (disks, SSDs, cloud volumes, etc) available 
-on the Kubernetes worker nodes to dynamically provision Kubernetes 
-Persistent Volumes. 
+Installing OpenEBS in your cluster is as simple as a few `kubectl` or `helm` commands depending on the storage engine. Here are the list of our Quickstart guides with detailed instructions for each storage engine.
```
Comment (ranjithwingrider) [inline docs/overview.md:63]: Installing OpenEBS in your cluster is as simple as a few `kubectl` or `helm` commands depending on the storage engine. -> Installing OpenEBS in your cluster is simple with the use of a few `kubectl` or `helm` commands depending on the storage engine that you are going to use.

Please ignore my comments if it is not simple.

**Review by ranjithwingrider — CHANGES_REQUESTED** [review]
@kmova Please check the review comments. Some typo and some suggestion for the betterment.

File: docs/overview.md (hunk 3 above)
Comment (kmova) [reply docs/overview.md:63]: the current text reads better, conversational. thanks for reading through everything!

**Review by ranjithwingrider — APPROVED** [review]

**Pull request merged**